deployment/injector.yml

```diff
@@ -1,6 +1,38 @@
 ---
-- hosts: injector
-  become: yes
-  roles:
-    - user-traffic-reader
-    - injector-configurator
+- hosts:
+    - hyperfoil_controller
+    - hyperfoil_agent
+  tasks:
+  - name: Install Java
+    package:
+      name: java-1.8.0-openjdk-headless
```

Comment:
does not work for ubuntu based workers

Reply:
Changes will be added to detect platform specific package names.

Reply:
```
$ ansible localhost -m setup|grep "ansible_distribution"
        "ansible_distribution": "Ubuntu",
        "ansible_distribution_file_parsed": true,
        "ansible_distribution_file_path": "/etc/os-release",
        "ansible_distribution_file_variety": "Debian",
        "ansible_distribution_major_version": "20",
        "ansible_distribution_release": "focal",
        "ansible_distribution_version": "20.04",
```

Please, do not resolve conversation until clearly solved or made it very clear, specially if you are asking something